build.gradle.kts

```diff
@@ -11,7 +11,7 @@ plugins {
 allprojects {
     repositories {
         mavenCentral()
-        maven(url = "https://dl.bintray.com/kotlin/kotlinx/")
+        jcenter()
```

Comment:
I didn't know Dokka needed Jcenter.

Reply:
Some of it's dependencies are currently only there, for the moment.